p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -160,10 +160,20 @@ Layout Description
         <ecc/>
     </section>
     <section>
-        <description>Hostboot Data (1.125M)</description>
-        <eyeCatch>HBD</eyeCatch>
+        <description>Hostboot Data - READ-ONLY (~819KB)</description>
+        <eyeCatch>HBD_RO</eyeCatch>
         <physicalOffset>0x305000</physicalOffset>
-        <physicalRegionSize>0x120000</physicalRegionSize>
+        <physicalRegionSize>0xCD000</physicalRegionSize>
+        <side>A</side>
+        <sha512Version/>
+        <readOnly/>
+        <ecc/>
+    </section>
+    <section>
+        <description>Hostboot Data - Read-Write (~333KB)</description>
+        <eyeCatch>HBD_RW</eyeCatch>
+        <physicalOffset>0x3D2000</physicalOffset>
+        <physicalRegionSize>0x53000</physicalRegionSize>
         <side>A</side>
         <sha512Version/>
```

Comment:
I am not sure it makes sense to version the RW  portion of this data.  It cannot be securel loaded

Reply:
Agreed.  Changed and tested the removal of <sha512version>